script.js

```diff
@@ -2,11 +2,71 @@
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  console.log(allEpisodes);
 }
-
+ const rootElem = document.getElementById("root");
+// show all episod lists in main page
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+   episodeList.forEach(episod=>{
+    const episodContainer= document.createElement("div");
+    const title=document.createElement("h3");
+    const image=document.createElement("img");
+    const summary=document.createElement("p");
+    episod.number < 10 && episod.season < 10
+      ? (title.textContent = `${episod.name}-S0${episod.number}E0${episod.season}`)
+      : (title.textContent = `${episod.name}-S${episod.number}E${episod.season}`);
+    image.src=`${episod.image.medium}`
+    summary.textContent = `${episod.summary}`.replace(/(<([^>]+)>)/gi, "");
+    episodContainer.appendChild(title);
+    episodContainer.appendChild(image);
+    episodContainer.appendChild(summary);
+    rootElem.appendChild(episodContainer);
+  });
 }
-
-window.onload = setup;
+  // search section
```

Comment:
I like how you're using comments to break up your sections, it makes your code nice & easy to follow